museek/plugin/sanity_check_observation_plugin.py

```diff
@@ -81,13 +91,16 @@ def run(self, scan_data: TimeOrderedData, all_data: TimeOrderedData, output_path
                                        f'Observation start time: {timestamp_dates[0]}\n ',
                                        f'\t \t and duration: {timestamp_dates[-1] - timestamp_dates[0]}'])
 
+        TA = TimeAnalysis(data=scan_data) 
```

Comment:
instances must be snake case, and please without abbreviations whenever possible